float64[6] rVal